src/ripple/protocol/impl/STLedgerEntry.cpp

```diff
@@ -78,7 +78,7 @@ void STLedgerEntry::setSLEType ()
     type_ = mFormat->getType ();
     if (!setType (mFormat->elements))
     {
-        if (auto j = debugLog())
+        if (auto j = debugLog().error())
```

Comment:
`Journal` is supposed to be lightweight and copyable. Nothing more than a pointer.